### PR TITLE
feat/kubectx-menu

### PR DESCRIPTION
**Pull request by mosheavni**
Create a float view for `:Kubectx`
image

### CODE REVIEW

**Comment by mosheavni** [comment]
ready for review @Ramilito

File: lua/kubectl/init.lua [hunk 1]
```diff
@@ -105,12 +106,13 @@ function M.setup(options)
 
   vim.api.nvim_create_user_command("Kubectx", function(opts)
     if #opts.fargs == 0 then
-      return
+      ctx_view.View()
```
Comment (Ramilito) [inline lua/kubectl/init.lua:109]: Nice!

File: lua/kubectl/mappings.lua [hunk 2]
```diff
@@ -33,6 +33,7 @@ function M.register()
   M.map_if_plug_not_set("n", "<C-f>", "<Plug>(kubectl.filter_view)")
   M.map_if_plug_not_set("v", "<C-f>", "<Plug>(kubectl.filter_term)")
   M.map_if_plug_not_set("n", "<C-n>", "<Plug>(kubectl.namespace_view)")
+  M.map_if_plug_not_set("n", "<C-x>", "<Plug>(kubectl.contexts_view)")
```
Comment (Ramilito) [inline lua/kubectl/mappings.lua:36]: Awesome mapping!

File: lua/kubectl/utils/viewsTable.lua [hunk 3]
```diff
@@ -16,6 +16,8 @@ return {
   services = { "services", "service", "svc" },
   configmaps = { "configmaps", "configmap", "cm" },
   crds = { "customresourcedefinitions", "crds", "crd", "customresourcedefinitions.apiextensions.k8s.io" },
+  namespace = { "namespace", "namespaces", "ns" },
```
Comment (Ramilito) [inline lua/kubectl/utils/viewsTable.lua:19]: While I like the idea of adding namespaces here, that would remove it from the fallback views and we wouldn't be able to edit/delete them

File: lua/kubectl/views/contexts/init.lua [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  M.contexts = vim.split(contexts, "\n")
+  return M.contexts
+end
+
+--- Change context and restart proxy
+--- @param cmd string
+function M.change_context(cmd)
+  local results = commands.shell_command("kubectl", { "config", "use-context", cmd })
+
+  vim.notify(results, vim.log.levels.INFO)
```
Comment (Ramilito) [inline lua/kubectl/views/contexts/init.lua:67]: This notify is making me have to press a key to continue, the results seem to be empty as well so I added a check for that and that works for me

File: lua/kubectl/utils/viewsTable.lua (hunk 3 above)
Comment (mosheavni) [reply lua/kubectl/utils/viewsTable.lua:19]: You are right! i'll remove it

Comment (mosheavni) [inline lua/kubectl/utils/viewsTable.lua:19]:
```suggestion
```

**Review by Ramilito — APPROVED** [review]

**Pull request merged**